src/components/Home.tsx

```diff
@@ -92,9 +92,6 @@ function TopComicComponent({ comicType }: { comicType: string }) {
             <div className="image-overlay h-full w-full rounded-lg bg-gradient-to-t from-plum absolute flex items-end justify-center text-center p-0 m-0">
               <p className="p-2 font-sans text-white">{comicRec.title}</p>
```

Comment:
Safety check here. I also recommend using nullish coalescing: `comicRec?.title ?? "PlaceHolderTitle"` to show a place holder title of some sorts if title is undefined or null. That's just a suggestion, you don't have to do it like that if it's not to your liking.

Reply:
Thank you. I will add safety checks and come back to nullish coalescing if I have time in the future.